rocketpy/plots/rocket_plots.py

```diff
@@ -327,39 +326,102 @@ def draw(self, vis_args=None):
                 linewidth=vis_args["line_width"],
             )
 
-        # TODO - Draw motor
+        # Draw motor
+        total_csys = self.rocket._csys * self.rocket.motor._csys
         nozzle_position = (
-            self.rocket.motor_position
-            + self.rocket.motor.nozzle_position
-            * self.rocket._csys
-            * self.rocket.motor._csys
+            self.rocket.motor_position + self.rocket.motor.nozzle_position * total_csys
         )
-        ax.scatter(
-            nozzle_position, 0, label="Nozzle Outlet", color="brown", s=10, zorder=10
+
+        nozzle = self.rocket.motor.plots._generate_nozzle(
+            translate=(nozzle_position, 0), csys=self.rocket._csys
         )
 
-        if isinstance(self.rocket.motor, (SolidMotor, HybridMotor)):
+        # List of motor patches
+        motor_patches = []
+
+        # Get motor patches translated to the correct position
+        if isinstance(self.rocket.motor, (SolidMotor)):
```

Comment:
Small code suggestion that does not affect funcionality: we could avoid the many `isinstances` and repeated code by catching `AttributeError` exceptions if the tank does not have any grains for example.

Reply:
I agree with the suggestion but I think we can refactor this later without causing any trouble. The code is working fine as it is right now, and any user/developer will understand that we created different conditions for each type of motor.

Thanks anyway.